bears/yaml/TravisLintBear.py

```diff
@@ -25,6 +27,31 @@ class TravisLintBear:
     CAN_DETECT = {'Formatting', 'Syntax'}
     SEE_MORE = 'https://docs.travis-ci.com/user/travis-lint'
 
+    @classmethod
+    def check_prerequisites(cls):
+        base_check = super().check_prerequisites()
+        if base_check is not True:
+            return base_check
```

Comment:
could you add a newline here? splits up the base check and the new check implemented by you :) I think it's really better to read with a newline here :+1: